ExportScripts/export_standalone_monolithic_windows.py

```diff
@@ -0,0 +1,285 @@
+#
+# Copyright (c) Contributors to the Open 3D Engine Project.
+# For complete copyright and license terms please see the LICENSE at the root of https://www.github.com/o3de/o3de.
+#
+# SPDX-License-Identifier: Apache-2.0 OR MIT
+#
+#
+import argparse
+import pathlib
+import logging
+import os
+import time
+import sys
+import subprocess
+import json
+
+from queue import Queue, Empty
+from threading  import Thread
+from typing import List
+from subprocess import Popen, PIPE
+
+logger = logging.getLogger('o3de.gamejam')
+LOG_FORMAT = '[%(levelname)s] %(name)s: %(message)s'
+logging.basicConfig(format=LOG_FORMAT)
+#This is an export script for MPS on the Windows platform
+# this has to be a complete standalone script, b/c project export doesnt exist in main branch yet
```

Comment:
This PR is going into development, not main. Did you mean for that?

Reply:
For now yes. I believe the O3DE contribution protocol calls for only commits processed in stabilization to go to main, which I would want at least this initial standalone script to eventually go. However currently project export does not exist in the main branch, so this is the closest compromise I could think of. Prepare and submit a standalone script, so that it can at lease be manually copied over.

Reply:
Main is an already a snapshot of the latest released version.
Since this change is after the release, it should go into the development and come out with the next release.

There is no need to try to backport functionality to an older release, plus it would require a stabilization period and support from the SIG-release group to try to create a point release(i.e 2205.1)